### PR TITLE
A Review - Part 1

### PR DESCRIPTION
**Pull request by adamrusted**
Various changes contributing to the great icon review.
Only one icon change, to update casing in the title.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -87,14 +87,14 @@
             "source": "https://www.ableton.com/en/legal/branding-trademark-guidelines/"
         },
         {
-            "title": "About.me",
-            "hex": "00A98F",
+            "title": "about.me",
+            "hex": "333333",
             "source": "https://about.me/assets"
```
Comment (service-paradis) [inline _data/simple-icons.json:92]: I think it could be added as guidelines too

File: _data/simple-icons.json [hunk 2]
```diff
@@ -190,68 +190,68 @@
         {
             "title": "Adobe After Effects",
             "hex": "9999FF",
-            "source": "https://www.adobe.com/products/aftereffects.html"
+            "source": "https://www.adobe.com/creativecloud.html"
```
Comment (service-paradis) [inline _data/simple-icons.json:193]: We can add https://developer.adobe.com/developer-distribution/creative-cloud/docs/guides/branding_guidelines/ for every Adobe icon as guidelines.

Maybe https://developer.adobe.com/distribute/cc-brand-assets should be used as source for some icons too.

**Review by service-paradis — CHANGES_REQUESTED** [review]
Only 2 small comments and it should be good to go!

**Comment by LitoMore** [comment]
Can we move `aboutdotme` icon change to another PR? I want to ignore these changes to our release note.

See https://github.com/simple-icons/simple-icons/pull/8776. PRs with `[skip]` prefixes will not be recorded.

In the meantime, I will change `[skip]` trick to a `skip release action` label in the future.

File: _data/simple-icons.json [hunk 3]
```diff
@@ -180,78 +181,92 @@
...
-            "source": "https://www.adobe.com"
+            "source": "https://www.adobe.com",
+            "guidelines": "https://developer.adobe.com/developer-distribution/creative-cloud/docs/guides/branding_guidelines/"
         },
         {
             "title": "Adobe Acrobat Reader",
             "hex": "EC1C24",
-            "source": "https://acrobat.adobe.com"
+            "source": "https://acrobat.adobe.com",
+            "guidelines": "https://developer.adobe.com/developer-distribution/creative-cloud/docs/guides/branding_guidelines/"
```
Comment (service-paradis) [inline _data/simple-icons.json:190]:
```suggestion
            "guidelines": "https://developer.adobe.com/developer-distribution/creative-cloud/docs/guides/branding_guidelines"
```

File: _data/simple-icons.json [hunk 4]
```diff
@@ -180,78 +181,92 @@
...
-            "source": "https://www.adobe.com/products/aftereffects.html"
+            "source": "https://www.adobe.com/creativecloud.html",
+            "guidelines": "https://developer.adobe.com/developer-distribution/creative-cloud/docs/guides/branding_guidelines/"
         },
         {
             "title": "Adobe Audition",
             "hex": "9999FF",
-            "source": "https://www.adobe.com/creativecloud/video.html"
+            "source": "https://www.adobe.com/creativecloud.html",
+            "guidelines": "https://developer.adobe.com/developer-distribution/creative-cloud/docs/guides/branding_guidelines/"
```
Comment (service-paradis) [inline _data/simple-icons.json:202]:
```suggestion
            "guidelines": "https://developer.adobe.com/developer-distribution/creative-cloud/docs/guides/branding_guidelines"
```

File: _data/simple-icons.json [hunk 5]
```diff
@@ -180,78 +181,92 @@
...
         },
         {
             "title": "Adobe Creative Cloud",
             "hex": "DA1F26",
-            "source": "https://www.adobe.com/creativecloud/plans.html"
+            "source": "https://www.adobe.com/creativecloud.html",
+            "guidelines": "https://developer.adobe.com/developer-distribution/creative-cloud/docs/guides/branding_guidelines/"
```
Comment (service-paradis) [inline _data/simple-icons.json:208]:
```suggestion
            "guidelines": "https://developer.adobe.com/developer-distribution/creative-cloud/docs/guides/branding_guidelines"
```

File: _data/simple-icons.json [hunk 6]
```diff
@@ -180,78 +181,92 @@
...
-            "source": "https://www.adobe.com/creativecloud/services.html"
+            "source": "https://www.adobe.com/creativecloud.html",
+            "guidelines": "https://developer.adobe.com/developer-distribution/creative-cloud/docs/guides/branding_guidelines/"
         },
         {
             "title": "Adobe Illustrator",
             "hex": "FF9A00",
-            "source": "https://www.adobe.com/products/illustrator.html"
+            "source": "https://www.adobe.com/creativecloud.html",
+            "guidelines": "https://developer.adobe.com/developer-distribution/creative-cloud/docs/guides/branding_guidelines/"
```
Comment (service-paradis) [inline _data/simple-icons.json:226]:
```suggestion
            "guidelines": "https://developer.adobe.com/developer-distribution/creative-cloud/docs/guides/branding_guidelines"
```

File: _data/simple-icons.json [hunk 7]
```diff
@@ -180,78 +181,92 @@
...
-            "source": "https://www.adobe.com/products/photoshop-lightroom-classic.html"
+            "source": "https://www.adobe.com/creativecloud.html",
+            "guidelines": "https://developer.adobe.com/developer-distribution/creative-cloud/docs/guides/branding_guidelines/"
         },
         {
             "title": "Adobe Photoshop",
             "hex": "31A8FF",
-            "source": "https://www.adobe.com/products/photoshop.html"
+            "source": "https://www.adobe.com/creativecloud.html",
+            "guidelines": "https://developer.adobe.com/developer-distribution/creative-cloud/docs/guides/branding_guidelines/"
```
Comment (service-paradis) [inline _data/simple-icons.json:250]:
```suggestion
            "guidelines": "https://developer.adobe.com/developer-distribution/creative-cloud/docs/guides/branding_guidelines"
```

File: _data/simple-icons.json [hunk 8]
```diff
@@ -180,78 +181,92 @@
...
-            "source": "https://www.adobe.com/ie/products/premiere.html"
+            "source": "https://www.adobe.com/creativecloud.html",
+            "guidelines": "https://developer.adobe.com/developer-distribution/creative-cloud/docs/guides/branding_guidelines/"
         },
         {
             "title": "Adobe XD",
             "hex": "FF61F6",
-            "source": "https://commons.wikimedia.org/wiki/File:Adobe_XD_CC_icon.svg"
+            "source": "https://helpx.adobe.com/support/xd.html",
+            "guidelines": "https://developer.adobe.com/developer-distribution/creative-cloud/docs/guides/branding_guidelines/"
```
Comment (service-paradis) [inline _data/simple-icons.json:262]:
```suggestion
            "guidelines": "https://developer.adobe.com/developer-distribution/creative-cloud/docs/guides/branding_guidelines"
```

**Review by service-paradis — CHANGES_REQUESTED** [review]

**Comment by adamrusted** [comment]
@LitoMore - do we still need the `[skip]` prefix if I'm moving the about.me update to another PR?

**Comment by LitoMore** [comment]
> Do we still need the `[skip]` prefix if I'm moving the about.me update to another PR?

We do. Icons will be listed under the `Updated icons` if you don't add the `[skip]` prefix.

But for the PR of `about.me` icon change, we should not add the `[skip]` prefix.

**Comment by adamrusted** [comment]
Ah, I've not been using the `[skip]` prefix for any of the other Great Icon Review PRs - and have already merged some.

**Comment by LitoMore** [comment]
@adamrusted No worries. Adding the `[skip]` prefix to the title before the weekly release then will be fine.

**Comment by adamrusted** [comment]
@LitoMore 
To the title of the PR, even if it's merged without it?

**Comment by LitoMore** [comment]
> @LitoMore To the title of the PR, even if it's merged without it?

Yes. Even if it's merged without it.

**Comment by LitoMore** [comment]
Remember to trim the `[skip]` prefix when merging it.

image

This field only specifies the commit message that merged to develop branch.

**Review by service-paradis — APPROVED** [review]

**Pull request merged**